README.md

```diff
@@ -1,2 +1,99 @@
 # testing
-Shared testing framework for Source projects
+Shared testing framework for Source projects.
+
+This repository provides a shared language for writing tests across Source Network projects.  Tests are composed entirely of `action.Action`s, `multiplier.Multiplier`s may be configured to mutate the declared actions to scale with various project-specific complexity multipliers.
+
+### Example consumption
+
+The following example taken from the Defra CLI package, and shows a simple test `TestSchemaAdd` along with the shared test executor for the test-package.
```

Comment:
todo: "..***is*** taken from..."

Reply:
Both current and proposed are correct, and are slightly different.  The current would probably benefit from a comma `The following example, taken from...`.

- [ ] Not sure which option I'll take, but I'll probably change this slightly

Reply:
It's `, and shows`.. that makes the lack of ***is/was*** more problematic.

Reply:
I don't understand why, you want the `, and shows` comma removed? Why?

Reply:
I didn't say that :)

Read this out loud
`The following example taken from the Defra CLI package, and shows a simple test...`

If you had instead
`The following example taken from the Defra CLI package shows a simple test...`
it would sound fine.

Alternatively my initial suggestion
`The following example was taken from the Defra CLI package, and shows a simple test...`
work with the `, and ...`

Reply:
> Alternatively my initial suggestion
The following example was taken from the Defra CLI package, and shows a simple test...
work with the , and ...

This is how it reads now.  I think we are on the same page.